docs/test/misc/ds6/alignment-controls-inline.html

```diff
@@ -75,13 +97,37 @@ <h2>Medium</h2>
             </div>
         </div>
     </span>
+    <span class="listbox">
+        <button class="expand-btn expand-btn--medium" aria-expanded="false" aria-haspopup="listbox">
```

Comment:
Looks like the `expand-btn--regular` needs some adjusting:

![image](https://user-images.githubusercontent.com/105656/46686567-7fe07f80-cbad-11e8-9839-ffe2c3139c0c.png)